kmip/tests/unit/core/attributes/test_attributes.py

```diff
@@ -498,25 +501,33 @@ def setUp(self):
             {'block_cipher_mode': BlockCipherMode.CBC,
              'padding_method': PaddingMethod.PKCS5,
              'hashing_algorithm': HashingAlgorithmEnum.SHA_1,
-             'key_role_type': KeyRoleType.BDK})
+             'key_role_type': KeyRoleType.BDK,
+             'digital_signature_algorithm':
+                DigitalSignatureAlgorithm.SHA256_WITH_RSA_ENCRYPTION,
+             'cryptographic_algorithm': CryptographicAlgorithm.HMAC_SHA512})
 
         self.cp_none = self.factory.create_attribute_value(
             AttributeType.CRYPTOGRAPHIC_PARAMETERS, {})
 
         # Symmetric key object with Cryptographic Parameters
-        # Byte stream edited to add Key Role Type parameter
+        # Byte stream edited to add:
+        #      Key Role Type parameter
+        #      Digital Signature Algorithm parameter
+        #      Cryptographic Algorithm parameter
```

Comment:
Hmmm, that's what I was worried about. I'll try to find examples in the other test case documents.

If I can't find any I'll go through the bytes manually and confirm they're valid.